src/generate-elements.ts

```diff
@@ -85,7 +86,7 @@ function serializeElement(
   const events = resolvedElement.events.map(
       e => ({
         name: e.name,
-        description: e.description,
+        description: e.description || '',
```

Comment:
can we leave as undefined if no description is found? It would make detection easier and less error prone downstream.